classes/item.rb

```diff
@@ -0,0 +1,22 @@
+class Item
+  attr_accessor :genre, :author, :label, :archived, :publish_date
```

Comment:
- [ ] in the Item class you should add the "has one" from the "one-to-many" relationship for Author, Label and genre, which means you have to add a method for each class which looks something like below:
```
    def genre=(genre)
      @genre = genre
      genre.items.push(self) unless genre.items.include?(self)
    end

```